crates/gitbutler-tauri/src/cli.rs

```diff
@@ -0,0 +1,89 @@
+use std::path::Path;
+
+use crate::error::Error;
+use anyhow::Context;
+use tracing::instrument;
+
+#[tauri::command(async)]
+#[instrument(err(Debug))]
+pub fn install_cli() -> anyhow::Result<(), Error> {
+    #[cfg(windows)]
+    {
+        // TODO
+        return Err(anyhow::anyhow!(
+            "CLI installation is not supported on Windows. Please install manually."
```

Comment:
Is the binary shipped on Windows? If so, then the CLI path could be mentioned here and users could help themselves. This would buy time to one day make it nicer, particularly if there is demand.